src/applications/disability-benefits/all-claims/utils/form0781.js

```diff
@@ -27,19 +27,15 @@ function combatOnlySelection(formData) {
  * @returns
  *   TRUE if
  *     - is set on form via the backend
- *     - Veteran is claiming a new disability
- *     - Veteran has selected connected condition choices on 'screener page'
+ *     - Veteran is claiming a new disability, and not a claim for increase
  *   else
  *     - returns false
  */
 export function showForm0781Pages(formData) {
-  const conditions = formData?.mentalHealth?.conditions || {};
   return (
     formData?.syncModern0781Flow === true &&
     isClaimingNew(formData) &&
-    Object.entries(conditions).some(
```

Comment:
This is the clause related to conditions selected on the Mental Health Screener page, which is getting removed